extensions/sql/contract-definition/schema/src/main/resources/org/eclipse/dataspaceconnector/sql/contractdefinition/schema/V0_0_0__Init_Database_Schema.sql

```diff
@@ -0,0 +1,37 @@
+--
+--  Copyright (c) 2022 Daimler TSS GmbH
+--
+--  This program and the accompanying materials are made available under the
+--  terms of the Apache License, Version 2.0 which is available at
+--  https://www.apache.org/licenses/LICENSE-2.0
+--
+--  SPDX-License-Identifier: Apache-2.0
+--
+--  Contributors:
+--       Daimler TSS GmbH - Initial SQL Query
+--
+
+-- table: edc_contract_definitions
+CREATE TABLE IF NOT EXISTS edc_contract_definitions
+(
+    contract_definition_id VARCHAR
+(
+    255
+) NOT NULL,
+    access_policy VARCHAR
+(
+    65535
```

Comment:
Instead of `VARCHAR(65535)` why not simply using `VARCHAR(MAX)`?

<sub> Denis Neuling <denis.neuling@daimler.com>, Daimler TSS GmbH, [legal info/Impressum](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md)